services/src/api/model/services.rs

```diff
@@ -70,6 +69,7 @@ pub struct MetaDataSuggestion {
 
 #[derive(Debug, Serialize, Deserialize, Clone, ToSchema)]
 #[serde(rename_all = "camelCase")]
+// TODO: validate user input
```

Comment:
I'm not sure if the TODO makes sense. You could only check for certain types of input errors, right? So you could check if the name is too long but not if it violates a duplicate contraint.

Reply:
Dataset names do not have to be unique. It doesn't really make sense to limit this globally, and as users can share datasets, there will be name clashes anyway.

Also we could limit the number of breakpoints in a symbology here etc. So I think it would make sense to validate here. As the PR is about getting the Task list working again, I didn't want to start completing our user input validation here though.